components/crx_file/crx_verifier_unittest.cc

```diff
@@ -175,6 +276,38 @@ TEST_F(CrxVerifierTest, ChecksRequiredKeyHashes) {
   EXPECT_EQ("UNSET", public_key);
 }
 
+#if defined(IN_MEMORY_UPDATES)
+TEST_F(CrxVerifierTest, ChecksRequiredKeyHashesForCrxFromString) {
+  const std::vector<uint8_t> hash;
+  std::string crx_str;
+  base::ReadFileToString(TestFile("valid_no_publisher.crx3"), &crx_str);
```

Comment:
Does it help to do EXPECT_TRUE(base::ReadFileToString(TestFile("valid_no_publisher.crx3"), &crx_str)) in all these tests? Just want to have a way to verify the result of base::ReadFileToString.

Reply:
Sure. I used ASSERT_TRUE in most of the tests, where only one test file is read from. Some tests actually contain multiple, independent tests and read from multiple files, and in these tests I used EXPECT_TRUE.

https://g3doc.corp.google.com/devtools/library_club/g3doc/totw/113.md